### PR TITLE
Update the required status checks for the addon controller

### PR DESCRIPTION
**Pull request by mprahl**
Related:
https://github.com/open-cluster-management-io/governance-policy-addon-controller/pull/15

### CODE REVIEW

File: core-services/prow/02_config/open-cluster-management-io/governance-policy-addon-controller/_prowconfig.yaml [hunk 1]
```diff
@@ -12,8 +12,11 @@ branch-protection:
                 required_approving_review_count: 1
               required_status_checks:
                 contexts:
-                - KinD tests (1.18, v1.19.16)
-                - KinD tests (1.18, latest)
+                - KinD tests / KinD tests (latest, true)
+                - KinD tests / KinD tests (latest, false)
+                - KinD tests / KinD tests (v1.19.16, true)
+                - KinD tests / KinD tests (v1.19.16, false)
+                - Linting and Unit tests / Linting and Unit tests
```
Comment (dhaiducek) [inline core-services/prow/02_config/open-cluster-management-io/governance-policy-addon-controller/_prowconfig.yaml:19]: I'm not sure why, but these all get identified inside GitHub with just the trailing name.
```suggestion
                - KinD tests (latest, true)
                - KinD tests (latest, false)
                - KinD tests (v1.19.16, true)
                - KinD tests (v1.19.16, false)
                - Linting and Unit tests
```

File: core-services/prow/02_config/stolostron/governance-policy-addon-controller/_prowconfig.yaml [hunk 2]
```diff
@@ -11,8 +11,10 @@ branch-protection:
                 dismiss_stale_reviews: true
               required_status_checks:
                 contexts:
-                - KinD tests (1.18, v1.19.16)
-                - KinD tests (1.18, latest)
+                - KinD tests / KinD tests (latest, true)
+                - KinD tests / KinD tests (latest, false)
+                - KinD tests / KinD tests (v1.19.16, true)
+                - KinD tests / KinD tests (v1.19.16, false)
```
Comment (dhaiducek) [inline core-services/prow/02_config/stolostron/governance-policy-addon-controller/_prowconfig.yaml:17]: Same comment as above. Won't the "Linting and Unit Tests" be coming midstream also?
```suggestion
                - KinD tests (latest, true)
                - KinD tests (latest, false)
                - KinD tests (v1.19.16, true)
                - KinD tests (v1.19.16, false)
                - Linting and Unit tests
```

Comment (dhaiducek) [inline core-services/prow/02_config/stolostron/governance-policy-addon-controller/_prowconfig.yaml:17]: Same comment as above. Won't the "Linting and Unit Tests" be coming midstream also?
```suggestion
                - KinD tests (latest, true)
                - KinD tests (latest, false)
                - KinD tests (v1.19.16, true)
                - KinD tests (v1.19.16, false)
                - Linting and Unit tests
```

**Comment by mprahl** [comment]
Thanks for the review @dhaiducek! I made the change in a rush and glad you got those mistakes.

**Review by dhaiducek — APPROVED** [review]

**Pull request merged**